docs/development/release_management.rst

```diff
@@ -8,23 +8,44 @@ must be done by people that have special privileges to do specific tasks
 but even if the **Release Manager** does not have those privileges, they should
 coordinate with the person that does to make sure the task is completed.
 
+In addition to the Release Manager, we typically recognize the following
+roles for a SecureDrop release:
+
+- **Deputy RM:** for additional time zone coverage, to delegate specific tasks,
+  and to act as backup in case of the RM becomes unavailable for any reason.
+
+- **Localization Manager:** to manage outreach to the translator community, and
+  to coordinate translation updates of existing strings.
+
```

Comment:
We've also been assigning a deputy LM, for similar reasons as the deputy RM.

Reply:
(Added)